bin/truffleruby

```diff
@@ -99,6 +122,9 @@ if [ $on_graalvm = false ]; then
 
 fi
 
+# no " to split $JAVA_OPTS into array elements
+java_args+=($JAVA_OPTS)
```

Comment:
Not sure if -cmd is used in JAVA_OPTS, but if it has -J prefix as a java option it should work from either of the sources.